### PR TITLE
Upgrading install profiles config following 8.8 core update

### PR DESCRIPTION
**Pull request by davidferlay**

### CODE REVIEW

File: web/profiles/sdd_core/config/install/system.theme.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 admin: seven
-default: bartik
+default: seven
```
Comment (andypost) [inline web/profiles/sdd_core/config/install/system.theme.yml]: Default is seven now?

File: web/profiles/sdd/config/install/page_manager.page_variant.panels_everywhere.yml [hunk 2]
```diff
@@ -88,7 +88,7 @@ variant_settings:
   storage_id: panels_everywhere
   builder: standard
   route_override_enabled: false
-  uuid: 1f1b2bda-8d73-4229-9fc4-6f51a576fa06
+  uuid: fc2d657c-de2b-4aa7-bcf3-2d42692dd1e6
```
Comment (andypost) [inline web/profiles/sdd/config/install/page_manager.page_variant.panels_everywhere.yml]: Interesting why this uuid changed

Comment (davidferlay) [reply web/profiles/sdd/config/install/page_manager.page_variant.panels_everywhere.yml]: Minor change, it tested ok

File: web/profiles/sdd_core/config/install/system.theme.yml (hunk 1 above)
Comment (davidferlay) [reply web/profiles/sdd_core/config/install/system.theme.yml]: Yes, cause bartik looks broken with panelseverywhere)

**Pull request merged**